### PR TITLE
Sync the model diagrams with the model repo

### PR DESCRIPTION
**Pull request by goneall**
This updates the diagrams with the model repo after merging https://github.com/spdx/spdx-3-model/pull/643

### CODE REVIEW

File: docs/images/model-security-profile.png
Comment (kestewart) [inline docs/images/model-security-profile.png:1]: Double check,  this is looking like the core, not security.   Also, is this reducnant with model Security.png.

**Review by kestewart — CHANGES_REQUESTED** [review]
Not sure why model-security-profile.png is included when there is a "model Secuirty.png".   Included by error?   Also file looks like Core+Software not security.

File: docs/images/model-security-profile.png
Comment (goneall) [reply docs/images/model-security-profile.png:1]: Removed the extra image.

I noticed this too while on the plane.

The image is present in the model repo - it is stale compared to model Security.png.

BTW - The image was not used in the website or PDF.

**Comment by goneall** [comment]
Resolved @kestewart feedback, merging

**Pull request merged**